ci-operator/jobs/openshift/cincinnati/openshift-cincinnati-master-postsubmits.yaml

```diff
@@ -0,0 +1,31 @@
+postsubmits:
+  openshift/cincinnati:
+  - agent: kubernetes
+    branches:
+    - ^master$
+    decorate: true
+    decoration_config:
+      skip_cloning: true
```

Comment:
I remember this caused issues when I had it in the presubmit job. @petr-muller can you verify that?

```suggestion
```

Reply:
See https://github.com/openshift/release/pull/3968 (which unfortunately doesn't have a given reason).

Reply:
@steveeJ Correct. Setting this causes Prow to not clone the repo, so that `dist/ghp_updater.sh` command would fail. You usually do not need `skip_cloning` unless you interact with the repo in some special way (like e.g. ci-operator does).